contracts/Governance/RadicleToken.sol

```diff
@@ -122,8 +122,8 @@ contract RadicleToken {
         return DECIMALS;
     }
 
-    /* @notice DOMAIN_SEPARATOR */
-    function DOMAIN_SEPARATOR() public view returns (bytes32) {
+    /* @notice domainSeparator */
+    function domainSeparator() public view returns (bytes32) {
```

Comment:
I think this is like, a convention / part of the API, so we should keep it in uppercase: https://github.com/ethereum/EIPs/blob/master/assets/eip-712/Example.sol#L35

Reply:
But on the other hand ERC-20's `decimals` always returns a constant, but is lowercase. Maybe the SCREAMING_SNAKE_CASE convention may not be best suited for the public parts of the API, there are no public fields anyway, just functions :thinking:

Does it need to be public? If it doesn't need to, then we don't need to bend to any conventions for future proofing.

Reply:
hmm maybe just drop a message on the radicle token channel

Reply:
I'd keep it public just in case, but maybe the case doesn't matter.

Reply:
in the EIP: https://eips.ethereum.org/EIPS/eip-712#definition-of-domainseparator it's actually camelCase..

Reply:
In https://eips.ethereum.org/EIPS/eip-2612#specification it's SCREAMING_SNAKE_CASE